src/features/query-bar/QueryBar.js

```diff
@@ -29,12 +29,13 @@ const QueryBar = ({
     onTemplateChange,
     templates,
 }: Props) => {
-    const nonItemColumns = columns && columns.filter(column => !ITEM_PROPERTIES.includes(column.property));
+    const columnsWithoutItems = columns && columns.filter(column => !ITEM_PROPERTIES.includes(column.property));
+    const columnsWithoutName = columns && columns.filter(column => column.property !== 'name');
```

Comment:
column.property is not unique e.g. the template may have a field where property="name". Please add column.source === 'item' (and confirm the exact value with Loren).